app/domain/authentication/authn_k8s/validate_pod_request.rb

```diff
@@ -11,7 +11,7 @@ module AuthnK8s
     # WebserviceNotFound, RoleNotAuthorizedOnWebservice, PodNotFound
     # ContainerNotFound, ScopeNotSupported, K8sResourceNotFound
 
-    ValidatePodRequest = CommandClass.new(
+    ValidatePodRequest ||= CommandClass.new(
```

Comment:
Were all of these `||=` lines added to fix the deprecation warnings? Maybe too late now but this seems like it could have been handled as a separate effort. It really pollutes the Rails 5 diff.

Reply:
I think they're all in one commit.  But, yeah, point taken.  Def the ship has sailed now though.